### ISSUE TITLE
Replace: All links at "50.7.74.29" 

### ISSUE THREAD

**Issue by lezhangisu** [iptv-org/iptv#6224]
### Channel Name

Japan TV channels

### What happened to the stream?

It is buffering to unplayable point

### Playlist

https://github.com/iptv-org/iptv/channels/jp.m3u

### Broken Link

All channels directed to ip 50.7.74.29

### Possible Replacement

NA

### Notes

The ip 50.7.74.29 is not ping-able now. It was good 5 hours ago. Is the server down? Anyone can fix it? Thanks

**Comment by Central302** [comment]
[Memo] All Port Not Work[Tine-Out]

**Comment by pencail** [comment]
Yes, the animax that I love to watch can't be watched either。ping 50.7.74.29，prompt timeout

**Comment by iant1000** [comment]
All Japanese channels with 50.7.74.29 are still not working. It would be amazing if you could fix it. Thanks in advance

**Comment by pShota** [comment]
drop my 2 cents confirm all 50.7.74.29 channels are dead

**Comment by lezhangisu** [comment]
I am able to ping the server 50.7.74.29 today. But streaming services still not available.

**Comment by Nobodynumber1** [comment]
Before had hacker to down services?

Today services had run again.

[Feb 02, 2022 11:43 PM UTC ]
Location Updated
[Feb 02, 2022 11:43 PM UTC ]
Routing Updated
[Feb 02, 2022 11:43 PM UTC ]
Orange
Service Added
 22/TCP/SSH

**Comment by UnderPM** [comment]
Stopped working again about 9h ago
